definitions/private-law/json/CaseTypeTab/CourtAdminTasklistTab/CaseTypeTab.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "LiveFrom": "25/11/2021",
+    "CaseTypeID": "PRLAPPS",
+    "Channel": "CaseWorker",
+    "TabID": "CATasks",
+    "UserRole": "caseworker-privatelaw-courtadmin",
+    "TabLabel": "CA Tasks",
+    "TabDisplayOrder": 0,
+    "CaseFieldID": "taskListReturnLabel",
+    "TabShowCondition": "[STATE] = \"AWAITING_SUBMISSION_TO_HMCTS\" OR [STATE] = \"AWAITING_RESUBMISSION_TO_HMCTS\"",
+    "TabFieldDisplayOrder": 1
+  },
+  {
+    "LiveFrom": "25/11/2021",
+    "CaseTypeID": "PRLAPPS",
+    "Channel": "CaseWorker",
+    "TabID": "CATasks",
+    "TabLabel": "CA Tasks",
+    "TabDisplayOrder": 0,
+    "CaseFieldID": "taskListReturn",
+    "TabFieldDisplayOrder": 2,
+    "FieldShowCondition": "taskListReturn = \"DO_NOT_SHOW\"",
+    "Comment": "taskListLabel is defined as '${taskList}' thus taskList must be present on tab"
```

Comment:
we dont need the return tab as the case cannot be returned to the courtadmin. It might be returned to solicitor in case they perform NoC. but we have that config already.

Otherwise, courtadmin can always amend the application. they wont return it.